### PR TITLE
task 8, patch reviews

### PR DESCRIPTION
**Pull request by Sazzo100**

### CODE REVIEW

File: __tests__/apis.test.js [hunk 1]
```diff
@@ -222,3 +222,80 @@ describe("POST a new comment to comments table", () => {
...
+    return request(app)
+    .patch("/api/reviews/skfuhsuif")
+    .send({ inc_votes: 'votehehe' })
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("votehehe is not a number");
+      });
+  });
+});
+
```
Comment (Slyose) [inline __tests__/apis.test.js:325]: This needs one more test to handle a missing request body. You have the error handling code for a missing inc_votes, but no corresponding test for it.

File: controllers/gamesC.js [hunk 2]
```diff
@@ -49,23 +50,41 @@ exports.getComments = (req, res, next) => {
...
-        next(err);
-      });
-  };
+  if (
+    !req.body.username ||
+    !req.body.body ||
+    Object.keys(req.body).length !== 2
+  ) {
+    next({ status: 400, msg: "incorrectly formatted comment body" });
+    return;
```
Comment (Slyose) [inline controllers/gamesC.js:59]: This error handling looks great, just needs a test!

**Review by Slyose — CHANGES_REQUESTED** [review]

File: controllers/gamesC.js (hunk 2 above)
Comment (Sazzo100) [reply controllers/gamesC.js:59]: you've responded to part of my task 7 code here but i've reworked it for the current task 8- should be good once i submit the pull request again!

File: __tests__/apis.test.js [hunk 3]
```diff
@@ -297,5 +297,29 @@ describe("8. PATCH /api/reviews/:review_id", () => {
...
+    return request(app)
+      .patch("/api/reviews/1")
+      .send({
+        inc_votes: '',
+      })
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("incorrectly formatted request body");
+      });
+  });
```
Comment (Slyose) [inline __tests__/apis.test.js:323]: The two new tests look great

File: controllers/gamesC.js [hunk 4]
```diff
@@ -74,6 +74,14 @@ exports.postComment = (req, res, next) => {
 exports.patchReview = (req, res, next) => {
   const { review_id } = req.params;
   const { inc_votes } = req.body;
+  if (
+    !req.body ||
+    !req.body.inc_votes ||
+    Object.keys(req.body).length !== 1
```
Comment (Slyose) [inline controllers/gamesC.js:80]: How come you've changed this to 1 from 2?

**Review by Slyose — APPROVED** [review]

**Pull request merged**